content/cloud-docs/workspaces/json-filtering.mdx

```diff
@@ -19,7 +19,7 @@ search and apply a filter at the same time.
 
 ## Entering a Filter
 
-Filters are entered by putting the filter in the aptly named **filter** box in
+Filters are entered by putting the filter in the apply named **filter** box in
```

Comment:
Ah! thank you for catching this. I think actually we can make it even shorter. How about this?

```suggestion
Enter a filter into the **filter** box in
```